tests/foreman/cli/test_capsule.py

```diff
@@ -40,7 +39,7 @@ def _make_proxy(request, options=None):
 
     @request.addfinalizer
     def _cleanup():
-        capsule_cleanup(proxy['id'])
+        Proxy.delete({'id': proxy['id']})
```

Comment:
Since this isn't the only use of the import I'll let it slide - but ideally this is also taking `target_sat` from the test function, just like in test_smartproxy.py.

Reply:
I'll change it, no reason to be inconsistent